recipes/perl-capture-tiny/0.34/build.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-
-# If it has Build.PL use that, otherwise use Makefile.PL
```

Comment:
I removed the perl packages that were empty place holders. I don't think I removed any others, but I was getting a message from the checker (at the bottom of the page) that I had conflicts with the master branch. 

I'll check it out.